api/service/explorer/service.go

```diff
@@ -258,6 +266,52 @@ func (s *Service) GetExplorerTransaction(w http.ResponseWriter, r *http.Request)
 	data.TX = *tx
 }
 
+// GetCommittee servers /comittee end-point.
+func (s *Service) GetCommittee(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	shardIDRead := r.FormValue("shard_id")
+	epochRead := r.FormValue("epoch")
+	shardID := uint64(0)
+	epoch := uint64(0)
+	var err error
+	if shardIDRead != "" {
+		shardID, err = strconv.ParseUint(shardIDRead, 10, 32)
+		if err != nil {
+			utils.Logger().Warn().Err(err).Msg("cannot read shard id")
+		}
+	}
+	if epochRead != "" {
+		epoch, err = strconv.ParseUint(epochRead, 10, 64)
+		if err != nil {
+			utils.Logger().Warn().Err(err).Msg("cannot read shard epoch")
+		}
+	}
+	if s.ShardID != uint32(shardID) {
+		utils.Logger().Warn().Err(err).Msg("incorrect shard id")
+		return
+	}
+	db := s.storage.GetDB()
+	bytes, err := db.Get([]byte(GetCommitteeKey(uint32(shardID), epoch)))
+	if err != nil {
+		utils.Logger().Warn().Err(err).Msg("cannot read committee")
+		return
+	}
+	committee := &types.Committee{}
+	if rlp.DecodeBytes(bytes, committee) != nil {
+		utils.Logger().Warn().Msg("cannot convert data from DB")
```

Comment:
Log the `error` returned by `DecodeBytes`.  Also make the message more specific, e.g. `"cannot decode committee data from DB"`.  Also return a 500 error.